extensions/Dockerfile

```diff
@@ -7,17 +7,17 @@ RUN mkdir /os
 WORKDIR /os
 ADD . .
 ARG OPENSHIFT_CI=0
-ARG VARIANT=""
 RUN if [ "${OPENSHIFT_CI}" != 0 ]; then ci/get-ocp-repo.sh --ocp-layer packages-openshift.yaml; fi
 RUN --mount=type=secret,id=yumrepos,target=/os/secret.repo <<EOF
 set -xeuo pipefail
 
-if [[ -n "${VARIANT}" ]]; then
-  MANIFEST="manifest-${VARIANT}.yaml"
-  EXTENSIONS="extensions-${VARIANT}.yaml"
+. /etc/os-release
+if [ $ID = rhel ]; then
+  MANIFEST="manifest-rhel-9.6.yaml"
+  EXTENSIONS="extensions-ocp-rhel-9.6.yaml"
```

Comment:
Does this imply these paths will need to be updated each time we say bump to the next version of RHEL? (i.e. RHEL 9.7 or 9.8 are here?) 

I would kind of prefer that we didn't have to remember to update these variables when that happens.